checkr/src/validators/candidate.ts

```diff
@@ -0,0 +1,36 @@
+import { z } from "zod";
+import { validateRequest } from "../middlewares/request-validator-middleware";
+
+const getCandidatesSchema = z.object({
+    // page: z.number().int().positive().optional(),
```

Comment:
I am not understanding, why is this commented

Reply:
Validators, I will add during API implementation. As of now, I am not sure completely of request body and validators may change.